articles/language/type-model.md

```diff
@@ -359,7 +360,7 @@ the following are true:
 - The operation `ConjugateInvertibleWith` may be invoked with an `inner`
   argument of either `Invertible` or `Unitary`.
 - The operation `ConjugateUnitaryWith` may be invoked with an `inner`
```

Comment:
Here, and in the previous sentence, we refer to `Conjugate...With` as *operations*, whereas in the code sample they are defined as *functions*. I'm unsure whether we should change it to say "function" here, or define as "operation" in the code sample. @cgranade thoughts?

Reply:
Yeah those should totally be functions, good catch. I have fixed that now.